doc_src/en/OmegaT_EditorsPanes.xml

```diff
@@ -729,14 +729,17 @@ When enabled, all formatting in the source segments are hidden.</programlisting>
 
     <para>When you enter a segment, the first fuzzy match (the one with the
     highest match percentage) is automatically selected. You can press
-    <keycombo><keycap>C</keycap><keycap>2, 3, 4, or 5</keycap>
-    </keycombo> to select a specific match, or cycle through the available matches with
-    <keycombo><keycap>C</keycap><keycap>↑ or ↓</keycap></keycombo>
-    (<keycombo><keycap>A</keycap><keycap>↑ or ↓</keycap></keycombo> on macOS).
-    Alternatively, you can use <link linkend="menus.edit" endterm="menus.edit.title"/>
+    <keycombo><keycap>C</keycap><keycap>2, 3, 4, or 5</keycap> </keycombo> to
+    select a different match. Alternatively, you can use <link
+    linkend="menus.edit" endterm="menus.edit.title"/>
```

Comment:
I think I prefer the original version's approach of keeping the keyboard related information together. It could be split into two sentences, with a change of paragraph for the menu alternative, if length was the main concern for the change:

```xml
<para>When you enter a segment, the first fuzzy match (the one with the
 highest match percentage) is automatically selected. You can press
<keycombo><keycap>C</keycap><keycap>2, 3, 4, or 5</keycap>
</keycombo> to select a specific match. You can also cycle through the available matches with
<keycombo><keycap>C</keycap><keycap>↑ or ↓</keycap></keycombo>
(<keycombo><keycap>A</keycap><keycap>↑ or ↓</keycap></keycombo> on macOS).</para>

<para>Alternatively, you can use <link linkend="menus.edit" endterm="menus.edit.title"/>
<link linkend="menus.edit.select.match" endterm="menus.edit.select.match.title"/>
to select a different match.</para>
```

Reply:
Ok.